torchmetrics/functional/text/rouge.py

```diff
@@ -240,7 +269,8 @@ def _rouge_score_compute(sentence_results: Dict[str, List[Tensor]]) -> Dict[str,
 
 def rouge_score(
     preds: Union[str, List[str]],
-    targets: Union[str, List[str]],
+    targets: Union[str, List[str], List[List[str]]],
+    accumulate: str = "best",
```

Comment:
```suggestion
    accumulate: Literal["avg", best"] = "best",
```

And `from typing_extensions import Literal`